images/build/Dockerfiles/gitHubRunners.BuildPackDepsBookworm.Dockerfile

```diff
@@ -0,0 +1,3 @@
+# https://github.com/actions/virtual-environments/blob/master/images/linux/Ubuntu1804-README.md
+# DisableDockerDetector "Need to determine correct way of handling hashes in mirror registry"
+FROM buildpack-deps:bookworm@sha256:af1d6cabbfa6bb5e8897a4fba5442451b77aecc090f35508a45d087a0a69eec2
```

Comment:
@cormacpayne, I believe he took the digest from here:
https://hub.docker.com/layers/library/buildpack-deps/bookworm/images/sha256-af1d6cabbfa6bb5e8897a4fba5442451b77aecc090f35508a45d087a0a69eec2?context=explore

But I am wondering why this specific digest is mentioned here. Why don't we take the latest one?

Reply:
That's correct, regarding the [SHA](https://hub.docker.com/layers/library/buildpack-deps/bookworm/images/sha256-af1d6cabbfa6bb5e8897a4fba5442451b77aecc090f35508a45d087a0a69eec2?context=explore). 

The benefit of keeping the SHA is for stability. I remember Oryx pipelines would fail more frequently due to changes on the latest referenced images. If either of you feel strongly about removing the SHA, I'm ok with removing it and seeing how the pipelines do. I tested locally and it worked.

Reply:
I think the SHA was originally coming from the `virtual-environments` repository so that we could be basing our images off of layers that were already cached on the GitHub Action runners to save users time when pulling the Oryx build images down (hence the `github-action` tag in our build image).

If I remember correctly, we used to manually update these SHAs as they were updated in the latest Ubuntu runners, so we _could_ automate the update of this SHA in the future, but I think that would be low-hanging fruit, so right now we shouldn't need to do anything.